common/src/main/scala/com/gu/media/aws/UploadAccess.scala

```diff
@@ -1,21 +1,42 @@
 package com.gu.media.aws
 
-import com.amazonaws.auth.{AWSStaticCredentialsProvider, BasicAWSCredentials}
 import com.amazonaws.services.securitytoken.AWSSecurityTokenServiceClient
+import com.amazonaws.services.stepfunctions.AWSStepFunctionsClient
+import com.amazonaws.services.stepfunctions.model.ListStateMachinesRequest
 import com.gu.media.Settings
+import scala.collection.JavaConverters._
 
 trait UploadAccess { this: Settings with AwsAccess =>
   val userUploadBucket: String = getMandatoryString("aws.upload.bucket")
   val userUploadFolder: String = getMandatoryString("aws.upload.folder")
   val userUploadRole: String = getMandatoryString("aws.upload.role")
 
+  val pipelineName: String = s"VideoPipeline$stage"
```

Comment:
I'd like to but unfortunately the cloud formation support for state machines doesn't let you set a name apart from the one directly in the YAML, which must be alphanumeric.